cmd/main.go

```diff
@@ -32,13 +32,13 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
 	declarativev1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/declarative/v1"
 	deployimagev1alpha1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/deploy-image/v1alpha1"
+	multimodulev1alpha1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/multi-module/v1alpha1"
 	golangv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
```

Comment:
Question:

> kubebuilder init --repo 
kubebuilder edit --plugins multi-module/v1-alpha --multimodule=true
What happens if we do not pass the flag?
What happens if we set false instead?

Reply:
if we do not pass the flag, it will count as false and it will disable it. This is due to
```
func (p *editSubcommand) BindFlags(fs *pflag.FlagSet) {
	fs.BoolVar(&p.multimodule, "multimodule", false, "enable or disable multimodule layout")
}
```
we could also default it to true, then we wouldnt need to pass the flag to enable it.

I think the flag is necessary to remove the multi-module support again when using edit. That way people can revert their changes. If we do not scope this in (e.g. by assuming people will revert by doing a git revert instead), then we can make the edit command easier, and I do not need to track the flag here.